### PR TITLE
chore: point stakepool and pgboss services to stakepool-v2 database

### PR DESCRIPTION
**Pull request by gytis-ivaskevicius**
# Context

_Reason for the change? If an issue exists, reference it here using a [keyword](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)_

# Proposed Solution

# Important Changes Introduced

### CODE REVIEW

**Comment by iccicci** [comment]
I suggest to rebase this on latest master to solve the failing e2e tests flow

**Comment by gytis-ivaskevicius** [comment]
Sounds like we are blocking rewards bug till the next release?

**Review by bernokl — APPROVED** [review]

**Comment by pparker** [comment]
This is connected to ROS ticket https://input-output.atlassian.net/browse/LW-10287

File: nix/cardano-services/deployments/default.nix [hunk 1]
```diff
@@ -431,7 +487,10 @@ in
 
         projectors = {
           handle.enabled = true;
-          stake-pool.enabled = true;
+          stake-pool = {
+            enabled = true;
+            env.PROJECTION_NAMES = lib.mkForce "stake-pool,stake-pool-metadata-job,stake-pool-metrics-job";
```
Comment (bernokl) [inline nix/cardano-services/deployments/default.nix:492]: Do we not want  "pool-rewards" for dev-preprod?

**Review by bernokl — APPROVED** [review]
LGTM, just open question on dev-preprod queues

File: nix/cardano-services/deployments/default.nix (hunk 1 above)
Comment (gytis-ivaskevicius) [reply nix/cardano-services/deployments/default.nix:492]: this is for live mainnet, made it compatible so we could merge this in

**Pull request merged**